src/facade/reply_builder.cc

```diff
@@ -277,7 +289,7 @@ void RedisReplyBuilder::SendBulkString(std::string_view str) {
   return Send(v, ABSL_ARRAYSIZE(v));
 }
 
-std::string_view RedisReplyBuilder::StatusToMsg(OpStatus status) {
+std::string_view SinkReplyBuilder::StatusToMsg(OpStatus status) {
```

Comment:
why did you change it? that's not good because these errors are not part of memcached protocol.

Reply:
That's not true: 

```
   949     if (cid->IsTransactional()) {       
   950       dist_trans.reset(new Transaction{cid});                                                     
   951                
   952       if (!dist_trans->IsMulti()) {  // Multi command initialize themself based on their mode.
   953         if (auto st = dist_trans->InitByArgs(dfly_cntx->conn_state.db_index, args_no_cmd);        
   954             st != OpStatus::OK)                
   955           return cntx->SendError(st);
   956       }                                                                                           
   957                                                                    
   958       dfly_cntx->transaction = dist_trans.get();
   959       dfly_cntx->last_command_debug.shards_count = dfly_cntx->transaction->GetUniqueShardCnt();   
   960     } else {                                                                                      
   961       dfly_cntx->transaction = nullptr;                                                           
   962     }                                                                        
   963   }  
```

Andddd 

`<< CI{"SET", CO::WRITE | CO::DENYOOM | CO::NO_AUTOJOURNAL`

So memcached commands are transactional. Same applies for `get` `incr` `decr` etc. If the weren't, our transactional system would be broken, because memcached commands would not be a part of a transaction whereas redis commands would

Reply:
`InitByArgs` always succeeds for these commands.

Reply:
We basically validate everything for memcached already at the parser level

Reply:
really bug prone tbh, some of the error reporting is already handled while some other doesn't(in case of this bug). Anyway, beyond the scope of this PR. I will see at some point how we can simplify this.

Reply:
I removed this for now

Reply:
actually no, we kinda need this because we need `SendError(ErrorReply)` which uses `status`. 

to be fair, it won't ever arrive to that block that uses status so I ended up removing this and send an internal error if the `ErrorReply` contains a status. You never know if we hit that so at least let's report this to be sure.